R/build_lookup.R

```diff
@@ -211,4 +234,6 @@ build_lookup_from_fastas <- function(data_dir, species) {
   utils::write.csv(lookup, file.path(save_dir, "lookup.csv"), row.names = FALSE)
```

Comment:
Not to change now unless you want to, probably open an issue for later: 

You're doing the same thing 3 times here, so probably worth a little function that does this write csv and has as inputs the data.frame to write and the `save_dir` path

Reply:
Added in [5967e8d](https://github.com/seshadrilab/tcrconvertr/pull/14/commits/5967e8d756c87a543ea30fd91ef6a450aa41f986), along with documentation and a test for it.